modules/api/src/main/java/org/apache/ignite/configuration/schemas/table/TableIndexConfigurationSchema.java

```diff
@@ -36,21 +36,21 @@
 
     /** Has default value flag. */
     @Value(hasDefault = true)
-    boolean uniq = false;
+    public boolean uniq = false;
 
     /** Columns configuration for sorted indexes. */
     @NamedConfigValue
     IndexColumnConfigurationSchema columns;
```

Comment:
This one has default values, we can't access package-private fields anymore because class loaders are different. I'd prefer making all fields public in another ticket.

Reply:
Moved to https://issues.apache.org/jira/browse/IGNITE-14788